app/src/main/java/com/wangdaye/mysplash/muzei/MysplashMuzeiArtSource.java

```diff
@@ -0,0 +1,99 @@
+package com.wangdaye.mysplash.muzei;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.util.Log;
+
+import com.google.android.apps.muzei.api.Artwork;
+import com.google.android.apps.muzei.api.RemoteMuzeiArtSource;
+import com.wangdaye.mysplash.common.data.entity.unsplash.Photo;
+import com.wangdaye.mysplash.common.data.service.PhotoService;
+import com.wangdaye.mysplash.photo.view.activity.PhotoActivity;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+import java.util.Random;
+
+import retrofit2.Call;
+import retrofit2.Response;
+
+/**
+ * Created by Sunng87 on 17-5-12.
+ */
+
+public final class MysplashMuzeiArtSource extends RemoteMuzeiArtSource {
+
+    private static final String SOURCE_NAME = "MysplashArtSource";
+
+    // 30 minutes
+    private static final long FAILED_RETRY_INTERVAL = 30 * 60 * 1000;
+
+    // 24 hours
+    private static final long UPDATE_INTERVAL = 24 * 60 * 60 * 1000;
+
+    private PhotoService photoService = PhotoService.getService();
+
+    /**
+     * Remember to call this constructor from an empty constructor!
+     *
+     */
+    public MysplashMuzeiArtSource() {
+        super(SOURCE_NAME);
+    }
+
+    private void exportPhoto(Photo photo) {
+        if (photo != null) {
+            Log.d(SOURCE_NAME, "Photo:" + photo.id);
+
+            Intent intent = new Intent(this, PhotoActivity.class);
+            intent.putExtra(PhotoActivity.KEY_PHOTO_ACTIVITY_ID, photo.id);
```

Comment:
Currently, start `PhotoActivity` from id doesn't work. I got this msg from logcat:

```
05-13 16:08:12.740 31806-31806/com.wangdaye.mysplash E/RecyclerView: No adapter attached; skipping layout
```